pkgs/development/tools/mold/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, lib, clang_12, cmake, lld_12, tbb, xxHash, zlib, openssl, git }:
+stdenv.mkDerivation rec {
+  pname = "mold";
+  version = "0.9.1";
+  src = fetchFromGitHub {
+    owner = "rui314";
+    repo = "mold";
+    rev = "v${version}";
+    sha256 = "sha256-yIkW6OCXhlHZ1jC8/yMAdJbSgY9K40POT2zWv6wYr5E=";
+  };
+  nativeBuildInputs = [ clang_12 cmake lld_12 tbb xxHash zlib openssl git ];
+  dontUseCmakeConfigure = true;
+  buildPhase = "make -j $NIX_BUILD_CORES";
+  installPhase = "mkdir -p $out $out/bin $out/share/man/man1 && PREFIX=$out make install";
+  meta = {
+    description = "mold is a high performance drop-in replacement for existing unix linkers.";
+    homepage = "https://github.com/rui314/mold";
+    license = lib.licenses.agpl3;
```

Comment:
We should ask wether this is agpl3Only or agpl3Plus according to https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206